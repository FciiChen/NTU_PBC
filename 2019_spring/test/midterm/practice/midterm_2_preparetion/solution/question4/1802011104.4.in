裕隆洋將桑尼（Sani Sakakini）在第1節就扭傷左腳踝，但他稍事休息後繼續拚戰，最後飆得全場最高30分、外帶11籃板5助攻3阻攻，助隊以83比79擊退衛冕軍璞園。桑尼展現超強毅力，直呼：「打籃球靠的是心智！」



桑尼的扭傷不輕，但他冰敷包紮過後選擇繼續奮戰。裕隆教練李啟億說：「我想把他換下來，他一直說不要，說他不行的時候會告訴我。」李啟億轉述桑尼的話，「他說你覺得痛的並不是腳，而是你的心跟頭腦，如果你覺得自己可以就會繼續拚。」



桑尼則回應：「沒錯，籃球就是看你的心智強弱，就算你只有1隻手也能打籃球！」桑尼表示，他雖然知道球季還很長，但他不想錯過任何一場比賽，因為他想幫助裕隆奪冠。



桑尼繳出亮眼數據，但完全不居功，「不管我得幾分都不重要，重點是球隊贏球，我們是個團隊，本土球員也都打得很好，這是場團隊勝利，同時也是為了教練贏的，因為他很信任我。」李啟億稱讚桑尼相當具有職業精神，讓大家都很感動。



璞園洋將麥克（Michael Holyfield）拿22分14籃板、瓊斯（Emanuel Jones）14分，吳家駿則有本土最高17分入袋，同時也是他生涯單場新高，無奈球隊吞敗。吳家駿表示：「球隊輸球，不在意自己得幾分，同時也還有很多地方要改進。」
BREAK