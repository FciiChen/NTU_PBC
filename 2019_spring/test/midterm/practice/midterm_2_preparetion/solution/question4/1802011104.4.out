同_也 3
李_億 3
1_分 2
1_籃 2
a_i 2
也_打 2
你_得 2
你_心 2
吳_駿 2
因_他 2
